hack/.ci/run-e2e-gke.sh

```diff
@@ -54,32 +54,64 @@ function kubectl_create {
 }
 
 function gather-artifacts {
-  kubectl -n=e2e run --restart=Never --image="${SO_IMAGE}" --labels='app=must-gather' --command=true must-gather -- bash -euExo pipefail -O inherit_errexit -c "function wait-for-artifacts { touch /tmp/done && until [[ -f '/tmp/exit' ]]; do sleep 1; done } && trap wait-for-artifacts EXIT && mkdir /tmp/artifacts && scylla-operator must-gather --all-resources --loglevel=2 --dest-dir=/tmp/artifacts"
+  kubectl_create -n=e2e -f=- <<EOF
+  apiVersion: v1
```

Comment:
nit: EOF wise, I think this should start at column 0

Reply:
done